internal/core/data/event_test.go

```diff
@@ -232,7 +233,7 @@ func TestAddEventWithValidationValueDescriptorDBError(t *testing.T) {
 
 func TestUpdateEventNotFound(t *testing.T) {
 	reset()
-	evt := models.Event{ID: bson.NewObjectId(), Device: "Not Found", Origin: testOrigin}
+	evt := models.Event{ID: bson.NewObjectId().Hex(), Device: "Not Found", Origin: testOrigin}
```

Comment:
If `github.com/satori/go.uuid` will replace `gopkg.in/mgo.v2/bson`, then all `bson.NewObjectId()` should be changed by `uuid.NewV4()`. Otherwise, don't use `github.com/satori/go.uuid`